### ISSUE TITLE
Conversion to INT8 fails

### ISSUE THREAD

**Issue by SM1991CODES** [NVIDIA-AI-IOT/torch2trt#810]
Hi,

I tried converting my model to int8 using the following:

`
model = model_small_fp32.BevDetNetSmall(in_channels=settings.N_CHANNELS_TRAIN_BEV, 
                                                out_kp_channels=settings.N_CHANNELS_PREDICTION_KP, 
                                                scale_H=2, scale_W=2, predict_3d_center=True).cuda()
    # load weights, set eval mode
    model.load_state_dict(torch.load(fp32_pth_path, map_location="cuda:0"))
    model.eval()

    dummy_x = torch.randn(2, settings.N_CHANNELS_TRAIN_BEV, settings.N_ROWS_TRAIN_BEV, settings.N_COLS_TRAIN_BEV).cuda()
    calib_datset = CalibDataset(path_calib_dataset_bev="/home/sambit/data/argoverse/2/val_bev/")

    # model_trt = torch2trt(model, [dummy_x], int8_mode=True,
    #                         int8_calib_dataset=calib_datset, 
    #                         max_batch_size=1, 
    #                         int8_calib_algorithm=trt.CalibrationAlgoType.ENTROPY_CALIBRATION_2)

    model_trt = torch2trt(model, [dummy_x], int8_mode=True, max_batch_size=2)

`

However, this fails with the following error message:

[10/09/2022-13:26:41] [TRT] [E] 2: [optimizer.cpp::getFormatRequirements::2945] Error Code 2: Internal Error (Assertion !n->candidateRequirements.empty() failed. no supported formats)

The same model works fine with FP16 and FP32.
As you can see above, I also tried with a calibration dataset - same error.

Please help.

Best Regards

**Comment by SrivastavaKshitij** [comment]
You will have to provide a working reproducible script

**Comment by gpastal24** [comment]
Hi I have been facing the same issue with torch2trt 0.0.4 and tensorrt >8.0.0. Using tensorrt 7.x and torch2trt 0.0.3 works fine
